src/Translator.php

```diff
@@ -16,11 +16,16 @@ class Translator extends LaravelTranslator {
      * @param  string  $locale
      * @return string
      */
-    public function get($key, array $replace = array(), $locale = null)
+    public function get($key, array $replace = array(), $locale = null, $fallback = true)
     {
         $result = parent::get($key, $replace, $locale);
         if($result === $key){
             $this->notifyMissingKey($key);
+
+            // Reget with fallback if true
+            if ($fallback) {
+                $result = parent::get($key, $replace, $locale, $fallback);
+            }
```

Comment:
Wouldn't this get the same result as before? Because the default is true anyways?